tests/integration/debug_files/bundle_jvm.rs

```diff
@@ -21,7 +21,7 @@ fn command_bundle_jvm_out_not_found_creates_dir() {
         testcase_cwd_path.join("jvm"),
     )
     .unwrap();
-    let _upload_endpoints = mock_common_upload_endpoints(ServerBehavior::Legacy);
+    let _upload_endpoints = mock_common_upload_endpoints(ServerBehavior::Legacy, vec![], None);
```

Comment:
It's fine for now, but keep in mind to maybe one day refactor it into an options struct, otherwise this args list can grooow and groow

Reply:
I refactored it, I think you're right that it's nicer with an options struct.